docs/API.md

```diff
@@ -21,11 +21,13 @@ There are a number of methods that a DB storage backend should implement:
     * .sessionTokenWithVerificationStatus(tokenId)
     * .sessionWithDevice(tokenId)
     * .deleteSessionToken(tokenId)
-    * .verifyToken(tokenVerificationId, accountData)
 * Key Fetch Tokens
     * .createKeyFetchToken(tokenId, keyFetchToken)
     * .keyFetchToken(id)
+    * .keyFetchTokenWithVerificationStatus(tokenId)
```

Comment:
I don't think so, no. A uniform API / the principle of least astonishment was my concern here.